graphlient.gemspec

```diff
@@ -17,4 +17,5 @@ Gem::Specification.new do |s|
   s.add_dependency 'faraday'
   s.add_dependency 'faraday_middleware'
   s.add_dependency 'graphql-client'
+  s.add_dependency 'mime-types'
```

Comment:
Oh, I haven't thought about it. 
Yup, `mini_mime` should work as expected too. 

And what do you think if we try to load mime types lib from users environment and if he does not have any mime types library then we throw an expection and enforce user to install mime types lib with a message_No mime types library detected you must add to your Gemfile `mime-types` or `mini_mime` gem_?
Code could look something like that: 
```ruby
def mime_type(file)
  if defined?(MIME::TYPES) 
    MIME::TYPES.type_for(file.path)
  if defined?(MiniMyme)
     MiniMyme.lookup_by_filename(file.path)
  else
    raise Graphlient::Errors::NoMimeTypesLibError, 'Please add `mime-types` or `mini_mime` gem to your Gemfile'
  end
end
```
 
It would nice for Rails users because Rails already includes `mime-types` gem.

Reply:
That sounds like a good idea 👍